packages/core/src/modules/connections/__tests__/ConnectionService.test.ts

```diff
@@ -63,457 +57,6 @@ describe('ConnectionService', () => {
-      expect(connection.mediatorId).toEqual('fakeMediatorId')
-      expect(connection.getTags()).toEqual(
-        expect.objectContaining({
-          verkey: connection.verkey,
-          invitationKey: recipientKey,
-        })
-      )
-      expect(connection.invitation).toMatchObject(invitation)
-      expect(connection.alias).toBeUndefined()
-      expect(connectionAlias.alias).toBe('test-alias')
-      expect(connection.theirLabel).toBe('test label')
-      expect(connection.imageUrl).toBe(connectionImageUrl)
-    })
-
-    it('returns a connection record with the autoAcceptConnection parameter from the config', async () => {
-      expect.assertions(3)
-
-      const invitation = new ConnectionInvitationMessage({
-        did: 'did:sov:test',
-        label: 'test label',
-      })
-
-      const connectionTrue = await connectionService.processInvitation(invitation, {
-        autoAcceptConnection: true,
-        routing: myRouting,
-      })
-      const connectionFalse = await connectionService.processInvitation(invitation, {
-        autoAcceptConnection: false,
-        routing: myRouting,
-      })
-      const connectionUndefined = await connectionService.processInvitation(invitation, { routing: myRouting })
-
-      expect(connectionTrue.autoAcceptConnection).toBe(true)
-      expect(connectionFalse.autoAcceptConnection).toBe(false)
-      expect(connectionUndefined.autoAcceptConnection).toBeUndefined()
-    })
-
-    it('returns a connection record with the alias parameter from the config', async () => {
-      expect.assertions(2)
-
-      const invitation = new ConnectionInvitationMessage({
-        did: 'did:sov:test',
-        label: 'test label',
-      })
-
-      const aliasDefined = await connectionService.processInvitation(invitation, {
-        alias: 'test-alias',
-        routing: myRouting,
-      })
-      const aliasUndefined = await connectionService.processInvitation(invitation, { routing: myRouting })
-
-      expect(aliasDefined.alias).toBe('test-alias')
-      expect(aliasUndefined.alias).toBeUndefined()
-    })
-  })
-
-  describe('createRequest', () => {
-    it('returns a connection request message containing the information from the connection record', async () => {
-      expect.assertions(5)
-      const connection = getMockConnection()
-
-      const { connectionRecord, message } = await connectionService.createRequest(connection)
-
-      expect(connectionRecord.state).toBe(ConnectionState.Requested)
-      expect(message.label).toBe(config.label)
-      expect(message.connection.did).toBe('test-did')
-      expect(message.connection.didDoc).toEqual(connection.didDoc)
-      expect(message.imageUrl).toBe(connectionImageUrl)
-    })
-
-    it('returns a connection request message containing a custom label', async () => {
-      expect.assertions(1)
-
-      const connection = getMockConnection()
-      const { message } = await connectionService.createRequest(connection, { myLabel: 'custom-label' })
```

Comment:
Those old are not. I forget to remove `createRequest`. There are two new ones with the `protocol` prefix. I'll look how easy it will be to reuse test for them, wich I originally wanted to do. The question is if would not be easy to do it together with moving those methods to `ConnectionProtocol` which I don't want to do now.

Reply:
It should be quite easy to reuse those tests. However, I'll create a new PR for it, I'll also add some checks for oob role and state.